src/mlx90632_extended_meas.c

```diff
@@ -282,19 +302,21 @@ int32_t mlx90632_set_meas_type(uint8_t type)
     int32_t ret;
     uint16_t reg_ctrl;
 
-    if ((type != MLX90632_MTYP_MEDICAL) & (type != MLX90632_MTYP_EXTENDED))
+    if ((type != MLX90632_MTYP_MEDICAL) & (type != MLX90632_MTYP_EXTENDED) & (type != MLX90632_MTYP_MEDICAL_BURST) & (type != MLX90632_MTYP_EXTENDED_BURST))
         return -EINVAL;
 
     ret = mlx90632_i2c_write(0x3005, MLX90632_RESET_CMD);
     if (ret < 0)
         return ret;
 
+    usleep(150, 200);
```

Comment:
wow, no? Somehow we need then something else, not a blunt usleep for this?

Reply:
Indeed, it does not look nice, but the other way would be to make some sort of dummy reads so that we 'waste' the 150 us. Perhaps something like a read loop (3-5 iterations) until we get a proper response from the device?